### ISSUE TITLE
Brainstorm name of this package

### ISSUE THREAD

**Issue by lwjohnst86** [science-collective/package#1]
A key to any good package is its name... and its also one of the hardest things to do! Think of some names and use the package and function `available::available()` to check if that package name already exists.

**Comment by Aastedet** [comment]
I set up a testing repo for the cite package, working title pubmedciteR:
https://github.com/Aastedet/pubmedciteR

I've cleared the major hurdles I think, and it's probably going to work out with a bit more work, so I should move it to the science-collective repo, right?

**Comment by lwjohnst86** [comment]
Yea, if you'd like to! It would make it significantly easier for me to contribute

**Comment by lwjohnst86** [comment]
Also, way to go, looks cool! Only thing we'd have to consider is that CRAN has a max 5 Mb package size. So you might have to get creative in ways to keep file size down.

**Comment by Aastedet** [comment]
Copied it here as-is, so very raw/exploratory.
A lot of the comments/mind-mapping should probably be GitHub-issues rather than bloating the documents/scripts, but there's a time for that later.
Also, the package-name can be whatever, I just picked something descriptive that contained a capital R (and was available). Can you rename a repo at some point, or how does that work?
Edit: That kind of relates to the starter post. I guess we'll end up working towards more than one package in separate repos inside /science-collective/.
Feel free to clean up my GitHub-ignorant mess as you see fit by the way - I have a very loose idea of what can/should be done here 😄

**Comment by lwjohnst86** [comment]
Yes we totally can rename the package. Alternatively, you can transfer ownership of the repository to here (go to settings, scroll down, in the Danger Zone area). I can give you access to do that on science-collective.

**Comment by Aastedet** [comment]
I figured there had to be a smarter way than copy/paste 😄
Now this repo is ahead of mine by a few minor edits, so let's just continue here (and maybe set up a new template repo for others to build other packages on).
For now I'm putting it on the backburner until december, when hopefully I have more time again. You and I could perhaps sit down and discuss it some time later.

Maybe we should also schedule a meeting in the group in january/february to keep everyone updated on what's been going on in the last year?
